roles/contiv_network/tasks/ovs.yml

```diff
@@ -53,9 +53,16 @@
     - "tcp:127.0.0.1:6640"
     - "ptcp:6640"
 
+- name: check selinux status
+  shell: sestatus | egrep "SELinux status:.*enabled"
+  register: selinux_set
+  when: (ansible_os_family == "RedHat")
```

Comment:
@gaurav-dalvi , please kick me if you want :)

Less code brings out one more readability enhancement that we shall do here.

I think we can simplify the task further by removing the `(ansible_os_family == "RedHat")` conditional altogether from this task and next task.